common/constants.py

```diff
@@ -43,6 +43,9 @@ class EntityKey:
     childcare_center_type = 'childcare.center_type'
     wholesale_market_app_type = 'wholesale.application_type'
     wholesale_market = 'wholesale.market'
+    population_zipcode_2020 = 'population.zipcode_2020'
```

Comment:
Follow naming convention for the other census population data we have in this file
Ex: `census_city_population`